ggml/src/ggml-cuda/pad_reflect_1d.cu

```diff
@@ -21,32 +21,33 @@ static __global__ void pad_reflect_1d_kernel_f32(
 
     const int64_t i3 = blockIdx.z;
     const int64_t i2 = blockIdx.y;
-    const int64_t i1 = blockIdx.x;
 
-    if (i1 >= ne01 || i2 >= ne02 || i3 >= ne03) {
+    const int64_t tile1  = blockIdx.x % ne01;     // i1
+    const int64_t tile0  = blockIdx.x / ne01;     // nth i0 tile
```

Comment:
Look into `fastdiv` defined in `common.cuh`. For such a small kernel this single division/modulo probably takes up a significant percentage of the runtime.